public/app/core/utils/richHistory.ts

```diff
@@ -146,19 +178,19 @@ export const mapNumbertoTimeInSlider = (num: number) => {
   let str;
   switch (num) {
     case 0:
-      str = 'today';
+      str = t('explore.history.rich-history-utils.today', 'today');
       break;
     case 1:
-      str = 'yesterday';
+      str = t('explore.history.rich-history-utils.yesterday', 'yesterday');
       break;
     case 7:
-      str = 'a week ago';
+      str = t('explore.history.rich-history-utils.aWeekAgo', 'a week ago');
```

Comment:
All names should follow the same naming convention (using dashes instead of camel case)

```suggestion
      str = t('explore.history.rich-history-utils.a-week-ago', 'a week ago');
```

Reply:
@ifrost Thank you for your guidance. It helped a lot for me.